gcp/table_gcp_bigquery_table.go

```diff
@@ -0,0 +1,217 @@
+package gcp
+
+import (
+	"context"
+	"strings"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+	"google.golang.org/api/bigquery/v2"
+)
+
+func tableGcpBigqueryTable(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "gcp_bigquery_table",
+		Description: "GCP Bigquery Table",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"dataset_id", "id"}),
+			Hydrate:    getBigqueryTable,
+		},
+		List: &plugin.ListConfig{
+			ParentHydrate: listBigQueryDatasets,
+			Hydrate:       listBigqueryTables,
+		},
+		Columns: []*plugin.Column{
+			// commonly used columns
+			{
```

Comment:
Can we add the important columns at the top, i.e. `table_id`, `dataset_id`, `self_link`, `id`?

Reply:
We are sorting the columns in alphabetic order? or as per important column?

Reply:
@ParthaI We generally follow the below order
* For general columns
- First 4-5 will be important columns,
- All other columns will be added in alphabetical order,
- Add all the `JSON` type columns at the end (in alphabetical order)
* For standard columns
- Columns needs to be added in following order (i.e. `title`, `tags`(if applicable) and `akas`)